WORKSPACE

```diff
@@ -46,9 +46,9 @@ load(
 
 git_repository(
     name = "io_bazel_stardoc",
-    commit = "4378e9b6bb2831de7143580594782f538f461180",
-    remote = "https://github.com/bazelbuild/stardoc.git",
-    shallow_since = "1570829166 -0400",
+    commit = "fa1878a1c4eae78a8aff016500777bedbc0bdf32",
+    remote = "https://github.com/segiddins/stardoc.git",
```

Comment:
Isn't bazelbuild master up to date with the segiddins branch? (Assuming this is true, because of https://github.com/bazelbuild/stardoc/compare/master...segiddins:master) Can we point to bazelbuild master instead?

Reply:
no, stardoc hasn't merged my PR yet